src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -182,8 +189,8 @@ public void setEmail(Email email) {
             this.email = email;
         }
 
-        public Optional<Email> getEmail() {
-            return Optional.ofNullable(email);
+        public Optional<Subject> getSubject() {
```

Comment:
Is `subject` optional? Your user guide says it is compulsory, so there is no need to wrap it in an `Optional` here.